src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
+
+  sourceString
+    .trim()
+    .split(';')
```

Comment:
Consider adding a check for an empty `sourceString` before proceeding with the split and forEach operations. This can prevent unnecessary operations and potential errors when the input string is empty or malformed.